README.md

```diff
@@ -12,12 +12,12 @@ As Hypervisor the following systems are supported:
 
 - List existing snapshots of a virtual machine.
 - Create a snapshot of a virtual machine.
-- Revert existing virtual machine to a previously created snapshot.
+- Revert existing virtual machines to a previously created snapshot.
 - Remove existing snapshots of a virtual machine.
 
 ## Installation
 
-See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin) for how to install Foreman plugins
+See [How_to_Install_a_Plugin](https://theforeman.org/plugins/#2.Installation) for how to install Foreman plugins
```

Comment:
We also have https://docs.theforeman.org/nightly/Managing_Hosts/index-katello.html#Creating_Snapshots_of_a_Managed_Host_managing-hosts.

I think it would be a good idea/small task to add the installation instructions to Foreman documentation if it's "just" running `foreman-installer ...`.

Reply:
Sure, why not. Not sure how to handle the not supported Debian package, though. Let me give the packaging a try... :hourglass:

Reply:
PR in foreman-documentation: https://github.com/theforeman/foreman-documentation/pull/2298

Reply:
PR in foreman-packaging: https://github.com/theforeman/foreman-packaging/pull/9576